README.md

```diff
@@ -506,6 +506,10 @@ To run the pipeline for JSONL files, the list of comma-separated headers also ne
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/CCRecords_sample.jsonl --headers=<comma_separated_list_of_headers>"
 ```
 
+Limitaions:
+1. The original schema of the input file will not be preserved as the data is simplified while converting it to DLP API 
```

Comment:
Since there's only one limitation, we don't need the 1.

Reply:
How about:

The original schema of the input file will not be preserved. This solution simplifies the data when it converts the data to a DLP API request object.

Reply:
Thanks @catherinerosesantos for all the corrections. I have pushed the new changes. Please review.